cordova-lib/spec-cordova/helpers.js

```diff
@@ -43,17 +54,101 @@ if (host.match(/win/)) {
 }
 */
 
-// Just use Android everywhere; we're mocking out any calls to the `android` binary.
-module.exports.testPlatform = 'android';
+module.exports.setEngineSpec = function (appPath, spec) {
+    appPath = getConfigPath(appPath);
+    var parser = new ConfigParser(appPath);
+
+    parser.removeEngine(module.exports.testPlatform);
+    parser.addEngine(module.exports.testPlatform, spec);
+    parser.write();
+};
+
+module.exports.getEngineSpec = function (appPath) {
```

Comment:
consider making this function more general by allowing the engine to be passed in as a separate parameter, instead of using the global variable module.exports.testPlatform.